### ISSUE TITLE
Message: unknown error: cannot connect to chrome at 127.0.0.1:9897

### ISSUE THREAD

**Issue by andriuy000** [ultrafunkamsterdam/undetected-chromedriver#884]
# I use profile path
options.add_argument("--user-data-dir=C:\\Users\\cod\\Desktop\\user_profile")
options.add_argument(f"--profile-directory=Profile {email_number}")

ERROR
Message: unknown error: cannot connect to chrome at 127.0.0.1:9897
from chrome not reachable
Stacktrace:
Backtrace:
	Ordinal0 [0x00E9ACD3+2075859]
	Ordinal0 [0x00E2EE61+1633889]
	Ordinal0 [0x00D2B680+571008]
	Ordinal0 [0x00D212EA+529130]
	Ordinal0 [0x00D4D96E+711022]
	Ordinal0 [0x00D46D24+683300]
	Ordinal0 [0x00D46B01+682753]
	Ordinal0 [0x00D7DBD3+908243]
	Ordinal0 [0x00D7D6AA+906922]
	Ordinal0 [0x00D77136+880950]
	Ordinal0 [0x00D4FEFD+720637]
	Ordinal0 [0x00D50F3F+724799]
	GetHandleVerifier [0x0114EED2+2769538]
	GetHandleVerifier [0x01140D95+2711877]
	GetHandleVerifier [0x00F2A03A+521194]
	GetHandleVerifier [0x00F28DA0+516432]
	Ordinal0 [0x00E3682C+1665068]
	Ordinal0 [0x00E3B128+1683752]
	Ordinal0 [0x00E3B215+1683989]
	Ordinal0 [0x00E46484+1729668]
	BaseThreadInitThunk [0x76F4FEF9+25]
	RtlGetAppContainerNamedObjectPath [0x77157BBE+286]
	RtlGetAppContainerNamedObjectPath [0x77157B8E+238]

**Comment by MattWaller** [comment]
See my comments on issue [870](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/870)

**Comment by Andrej-VB** [comment]
> See my comments on issue [870](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/870)

Yes: I've faced the same issue myself. It's probably due to crashing on startup or similar forced quitting. The easiest way to fix it is to empty the whole profile path folder.

**Comment by nuclear-bean** [comment]
To whoever might face this issue as well: I found out that this also occurs when you're trying to communicate with Chrome on a desktop-less environment when uc.Chrome was started without `headless=True` flag. 
So, make sure you did not forget it.
